### ISSUE TITLE
What is the difference between warp_ctc_pytorch.ctcloss and torch.nn.ctcloss?

### ISSUE THREAD

**Issue by arwagh** [meijieru/crnn.pytorch#238]
I noticed that this project and several more use warp_ctc_pytorch.ctcloss instead of torch.nn.ctcloss, however, installing warp_ctc_pytorch library did not work with me, so may I know what is the different between these two functions? does using torch.nn.ctcloss may raise some problems form me?

Thank you!

**Comment by xufuzhi** [comment]
the same question ....

**Comment by tiva-OG** [comment]
I hope my answer is still necessary. I believe the main difference is in the performance. **_warp_ctc_** performs the fastest when compared to the CTC Loss functions as described in the [README](https://github.com/baidu-research/warp-ctc/blob/master/README.md)
